strands_movebase/strands_movebase_params/costmap_common_params.yaml

```diff
@@ -3,17 +3,32 @@ robot_base_frame: /base_link
 
 map_type: voxel
 
-footprint: [[-0.30,-0.32], # Back side right
-            [-0.52,-0.11], # Back right
-            [-0.52, 0.11], # Back left
-            [-0.30, 0.32], # Back side left
-            [-0.05, 0.31], # Front side left
-            [ 0.05, 0.25], # Thick front left
-            [ 0.15, 0.13], # Thin front left
-            [ 0.20, 0.00], # Front
-            [ 0.15,-0.13], # Thin front right
-            [ 0.05,-0.25], # Thick front right
-            [-0.05,-0.31]] # Front side right
+footprint: [[-0.211159,-0.295000],
```

Comment:
Unfortunately an artifact of the simplification in Octave.... We could just fix it manually, the most important thing is that we keep the polygon points as close to the actual robots outline as possible. The old footprint was around 3.5cm too short in front!